neuraxle_tensorflow/tensorflow_v1.py

```diff
@@ -147,9 +147,9 @@ def fit_model(self, data_inputs, expected_outputs=None) -> BaseStep:
             feed_dict.update(additional_feed_dict_arguments)
 
         results = self.session.run([self['optimizer'], self['loss']], feed_dict=feed_dict)
-        self.loss.append(results[1])
+        self.losses.append(results[1])
```

Comment:
srry for the double comment here. I think this could even be `train_losses` or `test_losses`. Or have two variables for those losses if we compute both at some point. Just saying. It's okay to not rename for now, although in the doc this will need to be clear when doing a documentation pass.